### DISCUSSION TITLE
How to show pagination fixed on bottom?

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [quasarframework/quasar#14127] [General - Components / Directives / etc]
Hi,

How to show pagination fixed on bottom?

If you open my application:
https://sqlitedb.github.io/?f=https://sqlitedb-app.s3.amazonaws.com/files/northwind.db

And select a table. You will see that pagination is under table:

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/395096/182494293-7ade4445-1f00-4c96-925e-1728d4192455.png">

You need scroll to end of table to see pagination.

I want that pagination be fixed on bottom, like this:

![Untitled 2](https://user-images.githubusercontent.com/395096/182494793-0bee5544-bb84-459d-879d-5a158564c01f.jpg)

How can i do that?

**Comment by ewertonrfd** [comment]
Have you tried doing a custom pagination (https://quasar.dev/vue-components/table#pagination)
And leave the page with absolute position at the bottom of the page (screen)?
